doc/faq.rst

```diff
@@ -181,86 +181,60 @@ Using Spyder
-       |C:\\Anaconda3                       |   /opt/anaconda3                |
-       +------------------------------------+---------------------------------+
-       |C:\\Miniconda3                      |   /opt/miniconda3               |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Anaconda           |                                 |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Miniconda          |                                 |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Anaconda3          |                                 |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Miniconda3         |                                 |
-       +------------------------------------+---------------------------------+
+   If you want to use other packages in Spyder that don't come with our installer, you need to have your own Python distribution installed; we recommend `Miniconda`_ or another Conda-based option.
+   For Spyder to recognize it automatically, you should use a Conda-based distribution with its default install path.
 
    .. _Miniconda: https://docs.conda.io/en/latest/miniconda.html
 
-   Then, you need to create a new conda environment with the modules that you want to use with Spyder and include ``spyder-kernels`` in it. For example, if you want to use ``scikit-learn``, open your terminal or the Anaconda prompt on Windows and run the following commands:
-
-   .. code-block:: bash
-
-      conda create -n spyder-env -y
-      conda activate spyder-env
-      conda install spyder-kernels scikit-learn -y
-
-   Finally, you need to connect Spyder to this environment by changing Spyder’s default Python interpreter. To do this, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
+   Create a new conda environment with ``spyder-kernels`` and the packages that you want to use installed.
+   For example, if you want to use ``scikit-learn``, open your terminal (or Anaconda prompt on Windows) and run the following commands:
 
-   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section. Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment. If it is not listed, use the text box or the :guilabel:`Select file` button to enter the path to the Python interpreter you want to use.
+   .. code-block:: shell
 
-   **Your new environment will only be listed here if you installed Miniconda (or Anaconda) in the default path as shown in the table above.**
+      conda create -n scikit-learn-env -c conda-forge -y spyder-kernels scikit-learn
```

Comment:
```suggestion
      conda create -n scikit-learn-env -c conda-forge spyder-kernels scikit-learn
```
No big deal, but personally I prefer that example commands use as few flags as necessary and not get in the habit of using `-y`.

Reply:
Yeah, I have the same philosophy—I almost removed it as well, but had ended up leaving it how it was previously.